dts/arm/nordic/nrf52840.dtsi

```diff
@@ -137,6 +137,22 @@
 			interrupt-names = "wdt";
 			label = "WDT";
 		};
+
+		cryptocell: crypto@5002a000 {
+			compatible = "nordic,nrf-cc310";
+			reg = <0x5002A000 0x1000>;
+			label = "CRYPTOCELL";
+			status = "ok";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			cc310: crypto@5002b000 {
```

Comment:
@galak one question: could this cc310: be defined in the parent (arm) .dtsi file?